tests/src/Core/KeyValueStorage/DBKeyValueStorageTest.php

```diff
@@ -80,6 +80,6 @@ public function testUpdatedAt($k, $v)
 
 		$updateAtAfter = $entry['updated_at'];
 
-		self::assertLessThanOrEqual($updateAt, $updateAtAfter);
+		self::assertGreaterThanOrEqual($updateAt, $updateAtAfter);
```

Comment:
@annando @nupplaphil Found the source of the intermittent PHPUnit failures: the wrong order of parameters in this assertion. For `assertLessThanOrEqual`, the target high bound needs to be the first parameter. Instead of flipping the parameters which could still lead to confusion, I changed the assertion so that the order of parameters feels more natural.

We used the wrong assertion, but thanks to the "Equal" part it didn't systematically fail. In particular, the faster the test runner was, the harder it was to detect the logic error.